docs/azure-simple/README.md

```diff
@@ -4,80 +4,77 @@ This document walks through a Bedrock deployment.  It does not include everythin
+- [Create an RSA Key Pair to use as node key](#create-an-rsa-key-pair--to-use-as-node-key)
+- [Create an Azure Service Principal](#create-an-azure-service-principal)
   - [Configure Terraform For Azure Access](#configure-terraform-for-azure-access)
-  - [Clone the Bedrock Repository](#clone-the-bedrock-repository)
+- [Clone the Bedrock Repository](#clone-the-bedrock-repository)
   - [Set Up Terraform Deployment Variables](#set-up-terraform-deployment-variables)
   - [Deploy the Template](#deploy-the-template)
     - [Terraform Init](#terraform-init)
     - [Terraform Plan](#terraform-plan)
     - [Terraform Apply](#terraform-apply)
     - [Terraform State](#terraform-state)
-  - [Interact with the Deployed Cluster](#interact-with-the-deployed-cluster)
+- [Interact with the Deployed Cluster](#interact-with-the-deployed-cluster)
   - [Deploy an update using Kubernetes manifest](#deploy-an-update-using-kubernetes-manifest)
 
-## Prerequisites
+# Prerequisites
 
 Before starting the deployment, there are several required steps:
 
 - Install the required common tools (kubectl, helm, and terraform).  See also [Required Tools](https://github.com/microsoft/bedrock/tree/master/cluster). Note: this tutorial currently uses [Terraform 0.12.6](https://releases.hashicorp.com/terraform/0.12.6/).
 - Enroll as an Azure subscriber.  The free trial subscription does not support enough cores to run this tutorial.
 - Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
-- Set up [Flux manifest repository](#set-up-flux-manifest-repository).
-- Create an [Azure Service Principal](https://github.com/microsoft/bedrock/tree/master/cluster/azure/service-principal).
-- Create an [RSA key for logging into AKS nodes](#create-an-rsa-key-for-logging-into-aks-nodes). 
 
 The following procedures complete the prerequisites and walk through the process of configuring Terraform and Bedrock scripts, deploying the cluster, and checking the deployed cluster's health. Then we add a new manifest file to demonstrate Flux update.
 
-### Install the required tooling
+## Install the required tooling
 
-This document assumes one is running a current version of Ubuntu.  Windows users can install the [Ubuntu Terminal](https://www.microsoft.com/store/productId/9NBLGGH4MSV6) from the Microsoft Store.  The Ubuntu Terminal enables Linux command-line utilities, including bash, ssh, and git that will be useful for the following deployment.  
+This document assumes one is running a current version of Ubuntu.  Windows users can install the [Ubuntu Terminal](https://www.microsoft.com/store/productId/9NBLGGH4MSV6) from the Microsoft Store.  The Ubuntu Terminal enables Linux command-line utilities, including bash, ssh, and git that will be useful for the following deployment.  *Note: You will need the Windows Subsystem for Linux installe to use the Ubuntu Terminal on Windows*.
 
```

Comment:
Could you fix the typo "installe" to "installed" here?

Reply:
done